syntax = "proto3";

package Packet;

message DebugPacket {
	string debug_msg = 1;
}